llvm/projects/ton-compiler/cpp-sdk/tvm/tuple_list.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <tvm/cell.hpp>
+
+namespace tvm {
+
+class TupleList {
```

Comment:
Just out of curiosity - what's the use-case for TupleList?

Reply:
Added description for the class. Please ckeck it.